sql/moz-fx-data-marketing-prod/fetch/spend_alignment_by_campaign_v1/metadata.yaml

```diff
@@ -0,0 +1,13 @@
+description: >
+  Compare spend data by campaign between summary tables and detailed tables
+friendly_name: Campaign Spend Alignment
+labels:
+  incremental: false
+  schedule: weekly
+owners:
+  - bewu@mozilla.com
+scheduling:
+  dag_name: bqetl_marketing_fetch
+  date_partition_parameter: null
+  referenced_tables: []
```

Comment:
It would probably be best to list referenced tables here for correctness. They should still get ignored during dry runs. It's also possible to use wildcards in dataset and table names.

Reply:
I'll just remove it since the dry run works fine.  I think I just had it there to speed up testing.